charts/hono/values.yaml

```diff
@@ -1740,16 +1798,22 @@ amqpMessagingNetworkExample:
       # imageName contains the name (including tag) of the container
       # image to use for the example AMQP Messaging Network Broker
       imageName: "quay.io/artemiscloud/activemq-artemis-broker:1.0.6"
-      # livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+      # [DEPRECATED: use probes instead] livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
       # configuration property of the component's liveness probe.
+      # The value of the top level "livenessProbeInitialDelaySeconds" property will be used if not set.
       livenessProbeInitialDelaySeconds: 300
-      # readinessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+      # [DEPRECATED: use probes instead] readinessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
       # configuration property of the component's readiness probe.
+      # The value of the top level "readinessProbeInitialDelaySeconds" property will be used if not set.
       readinessProbeInitialDelaySeconds: 20
-      # resources contains the container's requests and limits for CPU and memory
-      # as defined by the Kubernetes API.
-      # Refer to https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
-      # for a description of the properties' semantics.
+      # probes allows you to overwrite the global probes config values for this specific component
+      probes:
+        livenessProbe:
+          periodSeconds: 17
+          timeoutSeconds: 3
+        readinessProbe:
+          periodSeconds: 10
+          timeoutSeconds: 3
```

Comment:
IMHO we should set the initialDelaySeconds in the probes here and leave the deprecated properties empty. That way we also provide an example of how to set the values in the desired way ...